README.md

```diff
@@ -0,0 +1,16 @@
+# Univ. of Cambridge Digital Hospitals webapp
+
+## Installation + build instructions
+
+```bash
+# Install latest version of node.js
+curl -sL https://deb.nodesource.com/setup_current.x | sudo bash -
```

Comment:
We can replace this with `setup_lts.x`.

Reply:
Still doesnt solve the issue that you cant run this command in windows